src/apiClient.ts

```diff
@@ -161,37 +180,44 @@ export default function APIClient(
         }
     };
 
-    this.sendSingleForwardingStatsToServer = function(forwardingStatsData) {
-        let url;
-        let data;
-        try {
-            const xhrCallback = function() {
-                if (xhr.readyState === 4) {
-                    if (xhr.status === 202) {
-                        mpInstance.Logger.verbose(
-                            'Successfully sent  ' +
-                                xhr.statusText +
-                                ' from server'
-                        );
-                    }
-                }
-            };
-            const xhr = mpInstance._Helpers.createXHR(xhrCallback);
-            url = mpInstance._Helpers.createServiceUrl(
-                mpInstance._Store.SDKConfig.v1SecureServiceUrl,
-                mpInstance._Store.devToken
-            );
-            data = forwardingStatsData;
+    this.sendSingleForwardingStatsToServer = async (
+        forwardingStatsData: IForwardingStatsData
+    ) => {
+        const { v1SecureServiceUrl: forwardingDomain} = mpInstance._Store.SDKConfig;
+        const { devToken } = mpInstance._Store;
+
+        const uploadUrl = `https://${forwardingDomain}${devToken}/Forwarding`;
+
+        let url: string;
+        const uploader: AsyncUploader = window.fetch
+            ? new FetchUploader(uploadUrl)
+            : new XHRUploader(uploadUrl);
+
+        const fetchPayload: fetchPayload = {
```

Comment:
I think we should tag this for refactor since I think a lot of the headers we're passing in are the same and can be set in the `AsyncUploader`

```suggestion
        // https://go.mparticle.com/work/SQDSDKS-6568
        const fetchPayload: fetchPayload = {
```